### PR TITLE
Update about cards

### PR DESCRIPTION
**Pull request by marymkohn**

### CODE REVIEW

**Review by mkhandker19 — APPROVED** [review]

File: pages/about.html [hunk 1]
```diff
@@ -46,66 +46,76 @@
...
-                <a href="https://www.linkedin.com/in/marymkohn/"><i class="fa-brands fa-linkedin fa-2xl"></i></a>
-                <a href="https://github.com/marymkohn"><i class="fa-brands fa-github fa-2xl"></i></a>
-                <a href="https://marymkohn.github.io/"><i class="fa-regular fa-globe fa-2xl"></i></a>
-                <a href="mailto:marymkohn@gmail.com"><i class="fa-solid fa-envelope fa-2xl"></i></a>
+                <aside>
+                    <a href="https://www.linkedin.com/in/marymkohn/"><i class="fa-brands fa-linkedin fa-2xl"></i></a>
+                    <a href="https://github.com/marymkohn"><i class="fa-brands fa-github fa-2xl"></i></a>
+                    <a href="https://marymkohn.github.io/"><i class="fa-regular fa-globe fa-2xl"></i></a>
+                    <a href="mailto:marymkohn@gmail.com"><i class="fa-solid fa-envelope fa-2xl"></i></a>
+                </aside>
```
Comment (marymkohn) [inline pages/about.html:68]: @stsalcido be sure to add your information here

File: pages/about.html [hunk 2]
```diff
@@ -46,66 +46,76 @@
...
```
Comment (marymkohn) [inline pages/about.html:101]: @mkhandker19 be sure to add your information here

File: pages/about.html [hunk 3]
```diff
@@ -46,66 +46,76 @@
...
```
Comment (marymkohn) [inline pages/about.html:118]: @cmdavis21 be sure to add your information here

File: js/Products.js [hunk 4]
```diff
@@ -39,8 +40,14 @@ export class Product {
...
-        const productDescription = document.createElement("p");
-        productDescription.textContent = this.description;
+        const productDescription = document.createElement("details");
+        const productSummary = document.createElement("summary");
+        const productDescriptionText = document.createElement("p");
+        productSummary.textContent = `Read about ${this.name}`;
+        productDescriptionText.textContent = this.description;
+
+        productDescription.appendChild(productSummary);
+        productDescription.appendChild(productDescriptionText);
```
Comment (marymkohn) [inline js/Products.js:50]: This creates a dropdown button for the description.

**Pull request merged**